### PR TITLE
[flutter_markdown] fix for latest pkg:markdown

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @srawlins @devoncarew

File: packages/flutter_markdown/test/table_test.dart [hunk 1]
```diff
@@ -441,8 +462,7 @@ void defineTests() {
               .toList();
           expect(text[0], '| abc | def | | --- | | bar |');
         },
-        // TODO(mjordan56): Remove skip once the issue #341 in the markdown package
-        // is fixed and released. https://github.com/dart-lang/markdown/issues/341
+        // TODO(kevmoo): Remove skip once min pkg:markdown > 6.0.1
```
Comment (kevmoo) [inline packages/flutter_markdown/test/table_test.dart:465]: A win here!

File: packages/flutter_markdown/lib/src/builder.dart [hunk 2]
```diff
@@ -470,9 +470,27 @@ class MarkdownBuilder implements md.NodeVisitor {
...
+        // `style` was using in pkg:markdown <= 6.0.1
+        // Can be removed when min pkg:markedwn > 6.0.1
         final String? style = element.attributes['style'];
         if (style == null) {
-          align = tag == 'th' ? styleSheet.tableHeadAlign : TextAlign.left;
+          // `align` is using in pkg:markdown > 6.0.1
+          final String? alignAttribute = element.attributes['align'];
+          if (alignAttribute == null) {
+            align = tag == 'th' ? styleSheet.tableHeadAlign : TextAlign.left;
+          } else {
```
Comment (kevmoo) [inline packages/flutter_markdown/lib/src/builder.dart:481]: I added a `_newMarkdown` flag in the test. We might-could do that here, too!

To discuss...

File: packages/flutter_markdown/test/table_test.dart (hunk 1 above)
Comment (domesticmouse) [reply packages/flutter_markdown/test/table_test.dart]: Any reason not to bump the minimum Markdown version now and clean this up as opposed to leaving this tech debt in the code base?

File: packages/flutter_markdown/lib/src/builder.dart (hunk 2 above)
Comment (domesticmouse) [reply packages/flutter_markdown/lib/src/builder.dart:481]: I’d prefer to bump minimum Markdown version and simplify the tests.

Comment (kevmoo) [reply packages/flutter_markdown/lib/src/builder.dart:481]: I *totally* understand. We'd like to do a major bump, but there is still more to do. And we can't roll to google3 unless things are passing.

**Review by domesticmouse — APPROVED** [review]
Clear for roll on the basis that @kevmoo is on the hook to tidy up tech debt in follow up PR.

**Pull request merged**